frontend/src/features/account/Login.tsx

```diff
@@ -104,4 +95,14 @@ const Login = () => {
   );
 };
 
+//check to see if user is using Internet Explorer
+//as their browser
+const usingIE = () => {
```

Comment:
Moved to bottom of file so that the component in this file is the first and primary function.